### PR TITLE
Remove dead OAuth code

### PR DESCRIPTION
**Pull request by claucambra**
Fixes #3692

### CODE REVIEW

**Comment by claucambra** [comment]
> @claucambra can you fix the compilation issues ?

Now fixed :)

File: src/gui/wizard/owncloudwizard.cpp [hunk 1]
```diff
@@ -19,14 +19,13 @@
...
-#include "wizard/owncloudhttpcredspage.h"
-#include "wizard/owncloudoauthcredspage.h"
+#include "wizard/flow2authcredspage.h"
 #include "wizard/owncloudadvancedsetuppage.h"
+#include "wizard/owncloudhttpcredspage.h"
+#include "wizard/owncloudsetuppage.h"
+#include "wizard/owncloudwizard.h"
 #include "wizard/webviewpage.h"
-#include "wizard/flow2authcredspage.h"
+#include "wizard/welcomepage.h"
```
Comment (mgallien) [inline src/gui/wizard/owncloudwizard.cpp:28]: would have been nicer not to change order here

File: src/libsync/creds/httpcredentials.cpp [hunk 2]
```diff
@@ -38,14 +38,13 @@ namespace OCC {
...
-    const char clientKeyPEMC[] = "_clientKeyPEM";
-    const char authenticationFailedC[] = "owncloud-authentication-failed";
-    const char needRetryC[] = "owncloud-need-retry";
+const char userC[] = "user";
+const char clientCertBundleC[] = "clientCertPkcs12";
+const char clientCertPasswordC[] = "_clientCertPassword";
+const char clientCertificatePEMC[] = "_clientCertificatePEM";
+const char clientKeyPEMC[] = "_clientKeyPEM";
+const char authenticationFailedC[] = "owncloud-authentication-failed";
+const char needRetryC[] = "owncloud-need-retry";
```
Comment (mgallien) [inline src/libsync/creds/httpcredentials.cpp:47]: please keep indentation changes in separate commits

File: src/gui/wizard/owncloudwizard.cpp (hunk 1 above)
Comment (claucambra) [reply src/gui/wizard/owncloudwizard.cpp:28]: Sorry, I have automatic clang-format enabled, will undo this change

File: src/libsync/creds/httpcredentials.cpp (hunk 2 above)
Comment (claucambra) [reply src/libsync/creds/httpcredentials.cpp:47]: auto clang-format again, will undo

File: src/gui/wizard/owncloudwizard.cpp (hunk 1 above)
Comment (claucambra) [reply src/gui/wizard/owncloudwizard.cpp]: done

File: src/libsync/creds/httpcredentials.cpp (hunk 2 above)
Comment (claucambra) [reply src/libsync/creds/httpcredentials.cpp]: done

**Review by allexzander — CHANGES_REQUESTED** [review]
@claucambra 

Compilation still fails:
`continuous-integration/drone/pr — Build is failing`

**Comment by mgallien** [comment]
@claucambra please fix the conflict ?

**Comment by claucambra** [comment]
> @claucambra please fix the conflict ?

now fixed
